index.android.js

```diff
@@ -22,6 +23,12 @@ var MusicControl = {
     NativeMusicControl.enableBackgroundMode(enable)
   },
   setNowPlaying: function(info){
+    //Check if we have an android asset id from react style image "require('./image.png')"
+    if (info.artwork && !isNaN(info.artwork)) {
+      info.artwork = resolveAssetSource(info.artwork).uri;
```

Comment:
Wouldn't be simpler if this was handled in the Android side? You just need to check if it's a ReadableMap and get the `uri` from it.